src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,35 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Is responsible for handling commands
+ */
+public class Command {
```

Comment:
I like how you have used a class to encompass all types of commands as this is neater and more scalable.

Reply:
I have updated my code base on your reviews, thank you!